pkg/kates/accumulator_test.go

```diff
@@ -58,6 +60,14 @@ func TestBootstrapNoNotifyBeforeSync(t *testing.T) {
 	// ConfigMaps prior to starting te Watch, all 10 of those ConfigMaps should be present in the
 	// first update.
 	assert.Equal(t, 10, len(snap.ConfigMaps))
+
+	t.Cleanup(func() {
```

Comment:
I don't think this is necessary is it? The client is being told to watch for configMaps and includes the LabelSelector as well. 

I guess you could have duplicate LabelSelectors so is that what this is trying to avoid?

Reply:
more the latter. this came about because when I originally wrote my other test I initially used the same label selectors so it was picking up way more config maps then it's supposed to. A future test may also accidentally pick them up

Also it's just good habit to clean up after yourself.